source/common/http/http1/codec_impl.cc

```diff
@@ -635,7 +635,8 @@ Http::Status ConnectionImpl::dispatch(Buffer::Instance& data) {
 
 Envoy::StatusOr<size_t> ConnectionImpl::dispatchSlice(const char* slice, size_t len) {
   ASSERT(codec_status_.ok() && dispatching_);
-  auto [nread, rc] = parser_->execute(slice, len);
+  auto nread = parser_->execute(slice, len).nread;
```

Comment:
This does not look like the right change. Before `execute` was called once, now it is called twice with the same data.

Reply:
Ohh thanks, Im looking into it. on running this with gdb i found out that it was failing at this line.